ui/src/components/admin/follow-up/BaptismFollowUpContainer.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
no need to import `React`

Reply:
I might be wrong but all react component files need import React even tho not explicitly being used

Reply:
oh i never do, but ok